wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,332 @@
+
+## 2021
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | *2019-10-06* | Present | [\[1\]](https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://osu.ppy.sh/users/4504101/) [\[3\]](https://ameobea.me/osutrack/user/whitecat/ "osu!track") [\[4\]](https://web.archive.org/web/20210119102542/https://old.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/ "Internet Archive — Reddit") [\[5\]](https://web.archive.org/web/20210203170702if_/https://www.youtube.com/watch?v=Bx4R7lovF-0 "Internet Archive — YouTube") |
+
+## Number of Reigns
+
+The following is a list of the the total amount of times a player on the previous list has reached the Rank #1 spot in their lifetime.
+
+| Player | Reigns |
+| --: | :-: |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 8 |
+| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 7 |
+| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 7 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 6 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 5 |
+| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 5 |
+| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 5 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 5 |
+| ![][flag_US] [Freddie Benson](https://osu.ppy.sh/users/7342622) | 4 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 4 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | 3 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | 3 |
+| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 2 |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2 |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | 2 |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2 |
+| ![][flag_DE] [ShadowSoul](https://osu.ppy.sh/users/494970) | 2 |
+| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | 2 |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | 2 |
+| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 1 |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | 1 |
+| ![][flag_US] [geckogates](https://osu.ppy.sh/users/252524) | 1 |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 1 |
+| ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | 1 |
+| ![][flag_KR] [Reimu-Desu](https://osu.ppy.sh/users/948713) | 1 |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 1 |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 1 |
+| ![][flag_KR] [sayonara-bye](https://osu.ppy.sh/users/713266) | 1 |
+| ![][flag_FI] [thelewa](https://osu.ppy.sh/users/475021) | 1 |
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 1 |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 1 |
+
+## Longest Combined Reign Length
+
+*Note: Only reigns after 2011-04-23 are included.*
```

Comment:
```suggestion
*Note: Only reigns after 23 April 2011 are included.*
```
[for some weird reason](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#date-and-time) the ASC only allows year-month-day in tables

Reply:
<insert themoreyouknow.gif here> 

This date may change depending on how well down we can get the earlier years, so just going to leave as is for the moment until we are sure as to how far back we can go.

Reply:
Resolving since no longer applicable to the current document.